aws-s3-transfer-manager/src/operation/download/chunk_meta.rs

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_sdk_s3::operation::get_object::GetObjectOutput;
+use aws_sdk_s3::operation::RequestId;
+use aws_sdk_s3::operation::RequestIdExt;
+
+/// request metadata other than the body that will be set from `GetObject`
+// TODO: Document fields
+#[derive(Debug, Clone, Default)]
+pub struct ChunkMetadata {
+    pub request_id: Option<String>,
+    pub extended_request_id: Option<String>,
```

Comment:
Any ideas why the request ID and request ID ext are traits and not just simple Strings? Should we also just implement those traits instead of exposing them as Strings?